.github/workflows/Helix-PR-Premerge-Check.yml

```diff
@@ -14,13 +14,31 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - name: Verify All Tasks are Finished
-        uses: venkatsarvesh/pr-tasks-completed-action@v1.0.0
+      - name: View context attributes
+        uses: actions/github-script@v3
         with:
-          repo-token: "${{ secrets.GITHUB_TOKEN }}"
-      - name: Verify the PR has been approved
-        uses: abinoda/label-when-approved-action@1.0.5
-        env:
-          APPROVALS: "1"
-          ADD_LABEL: "CheckedAndApproved"
-          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          script: console.log(context.payload.pull_request)
+      - name: Test
+        uses: actions/github-script@v3
+        with:
+          github-token: ${{secrets.GITHUB_TOKEN}}
+          script: |
+            const comments_url = context.payload.pull_request.review_comments_url
+            const result = await github.request(comments_url)
+            console.log(result)
+            for (const comment_info of result.data) {
+              const comment = await github.request(comment_info.url)
+              console.log(comment)
+            }
+      - name: Label PR if ready to be merged.
+        uses: actions/github-script@v3
```

Comment:
Have you considered the labeler: https://github.com/marketplace/actions/labeler

Reply:
Have you checked what it is used for? Do you think it fits our needs?
BTW, I don't think so...